workers/unity/Assets/Fps/Scripts/Editor/MapBuilderVisualisationWindow.cs

```diff
@@ -70,7 +70,23 @@ public void OnGUI()
                         SetupMapBuilder();
                     }
 
+                    var volumesPrefab = mapBuilderSettings.WorldTileVolumes == null
+                        ? null
+                        : Instantiate(mapBuilderSettings.WorldTileVolumes);
+
                     mapBuilder.CleanAndBuild(layerCount, seed);
+
+                    if (volumesPrefab != null)
+                    {
+                        if (Application.isPlaying)
```

Comment:
There is a helper utility for that that is used in the map builder as well. it will do this exact `if else` bit.

Reply:
Hmm I can't find it

Reply:
this one: https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.core/Utility/UnityObjectDestroyer.cs